WPMedia/fusion-news-theme-blocks
# THEMES-2015 Add editable headline, description and overline for extra-large-promo-block

**Be sure to run `npm test`, `npm run lint`, and write detailed test steps before requesting reviewers**

## Description

#### Jira Ticket: [THEMES-2015](https://arcpublishing.atlassian.net/browse/THEMES-2015)

Add editable headline, description and overline for extra-large-promo-block

## Test Steps

_Add detailed test steps a reviewer must complete to test this PR_

1. Checkout this branch `git checkout THEMES-2015`
2. Run fusion repo with linked blocks `npx fusion start -f -l @extra-large-promo-block`
3. Verify that the medium promo has an editable headline and description in PB Editor: http://localhost/pagebuilder/editor/curate?p=pRHgb9ScMtKyMDwOt&v=vnxm88raOnKyMDwOt

[THEMES-2015]: https://arcpublishing.atlassian.net/browse/THEMES-2015?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
